_posts/2024-10-28-matriz-riesgo-seccion-cliente.md

```
@@ -0,0 +1,22 @@
+title: Simplificando el Análisis: Nuevas Secciones en el KPI de cliente en la matriz de riesgo.
+subtitle: Matriz de Riesgo - Cliente.
+gh-repo: /capitalexpress/capitalexpress.github.io
+gh-badge: [star, fork, follow]
+tags: [ riesgo, creditos , clientes, ejecutivos, operaciones ]
+comments: false
+author: Erick Burgos
+---
+
+## Nuevas Secciones en el KPI de cliente
+
+
```

Markdownlint

Expected: 1; Actual: 2
Multiple consecutive blank lines

(MD012, no-multiple-blanks)